src/js/wand.js

```diff
@@ -50,12 +53,20 @@ var Wand = (function(wand) {
     wand.opts.nodes.forEach(function(node) {
 
       if (hasValidApiCallbackFns(node) === false) {
-        throw noSuchCallbackError;
+        wand.notifications.add({
+          text: 'We could not find a callback function that you asked for!',
+          type: 'alert'
+        });
+        throw wand.errors.noSuchCallbackError;
```

Comment:
I'd rather be [explicit rather than implicit](https://www.python.org/dev/peps/pep-0020/) when it comes to throwing errors.